detectors/built_in/custom_detectors/custom_detectors.py

```
@@ -1,41 +1,36 @@
+def slow_func(text: str) -> bool:
+    time.sleep(.25)
+    return False
+    
+from prometheus_client import Counter
 
+prompt_rejection_counter = Counter(
+    "trustyai_guardrails_system_prompt_rejections",
+    "Number of rejections by the system prompt",
+)
+  
+@use_instruments(instruments=[prompt_rejection_counter])  
```

Ruff (0.14.2)

Undefined name `use_instruments`

(F821)